### PR TITLE
[Docs][Connector-V2][Oss]Reconstruct the OssFile connector document

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Purpose of this pull request
Reconstruct the OssFile connector document follow the new format.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by hailin0** [comment]
please check ci error

**Comment by Hisoka-X** [comment]
> please check ci error

Done

**Comment by hailin0** [comment]
Please resolve the conflict

**Review by zhilinli123 — APPROVED** [review]

**Review by Carl-Zhou-CN — DISMISSED** [review]
Pending conflict resolution

**Comment by Hisoka-X** [comment]
> Pending conflict resolution

Done. Please take a look again. Thanks @Carl-Zhou-CN @zhilinli123 @hailin0

**Review by Carl-Zhou-CN — DISMISSED** [review]
LGTM

File: docs/en/connector-v2/source/OssFile.md [hunk 1]
```diff
@@ -279,11 +213,25 @@ Source plugin common parameters, please refer to [Source Common Options](common-
     endpoint = "oss-cn-beijing.aliyuncs.com"
     file_format_type = "orc"
   }
+}
 
+# Console printing of the read Oss data
+sink {
+  Console {
+    parallelism = 1
```
Comment (hailin0) [inline docs/en/connector-v2/source/OssFile.md]: remove

File: docs/en/connector-v2/source/OssFile.md [hunk 2]
```diff
@@ -298,18 +246,17 @@ Source plugin common parameters, please refer to [Source Common Options](common-
       }
     }
   }
+}
 
+# Console printing of the read Oss data
+sink {
+  Console {
+    parallelism = 1
```
Comment (hailin0) [inline docs/en/connector-v2/source/OssFile.md]: remove

File: docs/en/connector-v2/source/OssFile.md (hunk 1 above)
Comment (Hisoka-X) [reply docs/en/connector-v2/source/OssFile.md]: Done

**Review by Carl-Zhou-CN — APPROVED** [review]
+1

**Review by hailin0 — APPROVED** [review]

**Pull request merged**